Solves an issue for (Arm64) systems with 64 KiB pages where the build would fail with:
FAILED: [code=251] gen/v8/embedded.S snapshot_blob.bin
python ../../../../../../qt-everywhere-src-6.11.2/qtwebengine/src/3rdparty/chromium/v8/tools/run.py ./mksnapshot --turbo_instruction_scheduling --stress-turbo-late-spilling --target_os=linux --target_arch=arm64 --embedded_src gen/v8/embedded.S --predictable --no-use-ic --turbo-elide-frames --embedded_variant Default --random-seed 314159265 --startup_blob snapshot_blob.bin --no-native-code-counters --concurrent-builtin-generation --concurrent-turbofan-max-threads=0
Return code is -5

Solved upstream in:
https://chromium.googlesource.com/v8/v8/+/1e190bbb0396f7228797bf9b74cf689e83c5354e%5E%21
diff --git a/src/common/segmented-table.h b/src/common/segmented-table.h
index b0e31c4..924398e 100644
--- a/qt-everywhere-src-6.11.2/qtwebengine/src/3rdparty/chromium/v8/src/common/segmented-table.h
+++ b/qt-everywhere-src-6.11.2/qtwebengine/src/3rdparty/chromium/v8/src/common/segmented-table.h
@@ -41,16 +41,14 @@
   static constexpr bool kUseContiguousMemory = true;
   static constexpr size_t kReservationSize = size;
   static constexpr size_t kMaxCapacity = kReservationSize / kEntrySize;
-#if defined(V8_TARGET_OS_WIN) || defined(V8_HOST_ARCH_PPC64)
+#if defined(V8_TARGET_OS_WIN)
   // On windows the allocation granularity is 64KB and thus we cannot make a
   // segment smaller than that.
-  // PPC64 can utilize a 64KB page size.
   static constexpr bool kUseSegmentPool = false;
-  static constexpr size_t kSegmentSize = 64 * KB;
 #else
-  static constexpr bool kUseSegmentPool = true;
-  static constexpr size_t kSegmentSize = 16 * KB;
+  static constexpr bool kUseSegmentPool = kMinimumOSPageSize <= 16 * KB;
 #endif
+  static constexpr size_t kSegmentSize = kUseSegmentPool ? 16 * KB : 64 * KB;
 #else
   // On 32 bit, segments are individually mapped.
   static constexpr bool kUseContiguousMemory = false;
